extensions/binding/org.eclipse.smarthome.binding.openweathermap/ESH-INF/i18n/openweathermap_de.properties

```diff
@@ -60,14 +60,14 @@ thing-type.openweathermap.weather.label = Wetterinformationen
 thing-type.openweathermap.weather.description = Erm�glicht die Anzeige der aktuellen Wetterinformationen.
 
 thing-type.openweathermap.hourly-forecast.label = 3 Stunden Wettervorhersage
-thing-type.openweathermap.hourly-forecast.description = Erm�glicht die Anzeige einer 3 Stunden Wettervorhersage.
+thing-type.openweathermap.hourly-forecast.description = Aktiviert / Deaktiviert die automatische Erstellung einer 3 Stunden Wettervorhersage.
```

Comment:
I think in the heat of action you accidentally changed the wrong lines:

The 

```
bridge-type.config.openweathermap.weather-api.hourlyForecast.description
...
bridge-type.config.openweathermap.weather-api.dailyForecast.description
```

are the ones describing the config parameter of the bridge.

Reply:
:sweat: Ups, in the lunch break between two meetings ... I am eager to get this finally merged ...